haiku-games/bebattle/bebattle-1.0.0.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A fun strategy board game"
+DESCRIPTION="
```

Comment:
Start the DESCRIPTION here ... (remember the 80 char limit)

Reply:
No indent on the next line, @humdingerb  maybe you can comment? ;)

Reply:
Some time ago the style was different, but not all recipes have had their style updated.
Actually, when new recipes are added, we really require that the new style be used.
On old recipes, when they get updated for a "valid" reason, sometimes they also have their DESCRIPTION style updated, sometimes not. It depends on the mood of the contributor ;)
Well, the rationale is more or less that for these old recipes we think it's OK to keep the old style, because it does not make much difference.